server/src/index.ts

```diff
@@ -166,6 +174,8 @@ const typeDefs = gql`
     appLogs(appId: String!): AppLogsResult!
     databaseInfo(databaseId: String!): DatabaseInfoResult!
     databaseLogs(databaseId: String!): DatabaseLogsResult!
+    databasesLinkedToApp(appId: String!): DatabasesLinkedToAppResult!
```

Comment:
I am wondering if it should rather be a subset od apps eg:
```graphql
app(id: "") {
  id
  name
  databases {
	id
  }
}
```
what do you think?

Reply:
that way we can do it in just 1 query

Reply:
@pradel this makes a lot of sense, same also could be done for the db query. With this we will be able to get rid of two extra queries. I Will refactor in this PR.